pkg/transport/listener_test.go

```diff
@@ -401,3 +453,21 @@ func TestIsClosedConnError(t *testing.T) {
 		t.Fatalf("expect true, got false (%v)", err)
 	}
 }
+
+func TestSocktOptsEmpty(t *testing.T) {
+	tests := []struct {
+		sopts SocketOpts
+		want  bool
+	}{
+		{SocketOpts{}, true},
+		{SocketOpts{ReuseAddress: true, ReusePort: false}, false},
+		{SocketOpts{ReusePort: true}, false},
```

Comment:
Perhaps this should be added to the test list:
```
{SocketOpts{ReuseAddress: false, ReusePort: false}, true},
```

Reply:
We cover that in the first case right when the struct is initialized. https://play.golang.org/p/AR3W-23YAZq

Reply:
Yeah. I was just trying to cover an explicit case of setting both to `false`, but I agree it is the same case.